airbyte-integrations/connectors/source-smartsheets/setup.py

```diff
@@ -11,6 +11,6 @@
     author="Nate Nowack",
     author_email="contact@airbyte.io",
     packages=find_packages(),
-    install_requires=["airbyte-cdk~=0.1", "pytest==6.1.2", "smartsheet-python-sdk==2.105.1"],
+    install_requires=["airbyte-cdk~=0.1", "pytest==6.1.2", "pytest-mock~=3.6.1", "smartsheet-python-sdk==2.105.1"],
```

Comment:
maybe it's worth to move all testing libraries to `extras_require` section of `setup.py` ?